src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -391,13 +397,14 @@ def delete_connection_strings(resource_group_name, name, setting_names, slot=Non
 
 
 CONTAINER_APPSETTING_NAMES = ['DOCKER_REGISTRY_SERVER_URL', 'DOCKER_REGISTRY_SERVER_USERNAME',
-                              'DOCKER_REGISTRY_SERVER_PASSWORD', 'DOCKER_CUSTOM_IMAGE_NAME']
```

Comment:
I might be out of context, but why we remove this?

Reply:
We want to solely use linuxfxversion instead of this appsetting.

Reply:
This will be a breaking change, but I am fine to proceed. The reason I asked the question is the `set` command still takes `docker_custom_image_name`, so hiding it from `show` command breaks the e2e flow. My suggestions are:
1.  remove the `docker_custom_image_name` across all the commands and replace it with `linuxfxversion`. This cleans up the code base, very good thing
2.  discuss within your team and settle on using `linuxfxversion` moving forward, personally I feel the name is pretty weird, so would be good we can expose a different term to end users
3. I will call out the breaking change in the readme. Webapp/linux command has never been GA'd , so a few controlled breaking change is fine. We can also first emit out a warning for one sprint and then pull the plug later